provider/cmd/pulumi-resource-cloud-toolkit-aws/databases/auroraMysqlArgs.ts

```diff
@@ -0,0 +1,186 @@
+import * as pulumi from "@pulumi/pulumi";
+
+export interface AuroraMysqlArgs {
+  /**
+   * Version for database
+  */
+  version?: AuroraMysqlVersion;
+ 
+  /**
+  * Configuration parameters for the database
+  */
+  database?: AuroraMysqlDatabaseArgs;
+ 
+  /**
+   * The instance type for the cluster
+   */
+  instanceType?: pulumi.Input<string>;
+ 
+  /**
+   * The number of instances to be created for Aurora cluster
+   */
+  instancesCount?: number;
+ 
+   /**
+   * Logging configuration parameters for Aurora cluster
+   */
+   logging?: AuroraMysqlLoggingArgs;
+
+   /**
+   * Monitoring configuration parameters for Aurora cluster
+   */
+   monitoring?: AuroraMysqlMonitoringArgs;
+ 
+   /**
+   * Networking configuration parameters for Aurora cluster
+   */
+   networking?: AuroraMysqlNetworkingArgs;
+ 
+   /**
+   * Backup configuration parameters for Aurora cluster
+   */
+   backup?: AuroraMysqlBackupArgs;
+}
+
+/**
+* Set of allowed versions for the database
+*/
+export enum AuroraMysqlVersion {
+    V8_0 = "8.0",
+    V5_7 = "5.7",
+    V5_6 = "5.6"
```

Comment:
We only support 5.7 or 8.0 in RDS. I think that for consistency between component maintenance there should be feature parity.

Reply:
Fixed!